tests/ui/derivable_impls.stderr

```diff
@@ -23,10 +27,14 @@ LL | |     }
 LL | | }
    | |_^
    |
-   = help: try annotating `TupleDefault` with `#[derive(Default)]`
+   = help: Remove it
+help: and instead derive it
+   |
+LL | #[derive(Default)]
+   |
```

Comment:
Since the `Span` is 0-length long, the `struct XXX` part is not shown. I don't think this is a big issue now that it can be fixed automatically, but didn't find how to fix it in case it's a deal breaker.